src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -96,42 +165,175 @@ std::array<tt::remove_cvref_wrap_t<T>, Dim> Translation<Dim>::frame_velocity(
              << velocity.size()
              << ") does not match the dimension of the translation map (" << Dim
              << ").");
+  // Section for handling overloaded constructor
+  tt::remove_cvref_wrap_t<DataType> radial_function_value(
```

Comment:
This is almost identical to the `operator()` I think except for the function of time gets the deriv instead of the function. Maybe factor this out into a free function that just takes either 0 or 1 for the function value vs derivative.